tests/nlu/utilities.py

```diff
@@ -11,6 +11,15 @@ def write_file_config(file_config):
         return f
 
 
+def verify_sequence_not_none(sequence, sentence):
+    if sequence and sentence:
```

Comment:
You could write this as
```python
return (
    sequence.features if sequence else None,
    sentence.features if sentence else None,
)
```
This would also cover the case where both are `None`.